### ISSUE TITLE
Visual Studio ignoring CPM opt-out when ManagePackageVersionsCentrally is false but Directory.Packages.props exists

### ISSUE THREAD

**Issue by wazzamatazz** [NuGet/Home#12359]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

VS 2022 Enterprise 17.4.3

### Impact

It's more difficult to complete my work

### Repro Steps & Context

Consider a repository with the following structure:

```
- Directory.Packages.props
- src/
  - Directory.Packages.props
  - solution.sln
  - project/
    - project.csproj
 ```

- The top-level `Directory.Packages.props` opts into central package management (`<ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>`). 
- The nested `Directory.Packages.props` file in the `src` folder opts out of central package management (`<ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>`). 
- When `solution.sln` is opened in Visual Studio and you add a package reference to the project via the VS Package Manager UI, the project reference in the project file is added without a version and a package version is added to the nested `Directory.Packages.props` instead, even though the closest `Directory.Packages.props` file to the solution has opted out of central package management.

Steps to reproduce:

1. Download and unzip the attached file: [cpm_issue.zip](https://github.com/NuGet/Home/files/10393156/cpm_issue.zip)
2. Open the solution file in Visual Studio.
3. Add a NuGet package reference to the project via the Package Manager UI.

Workaround is to manually enter or update package references in the project file instead of using the UI.

**Comment by jeffkl** [comment]
@tmeschter I've confirmed this behavior.  Does Project System look at the `ManagePackageVersionsCentrally` property?  Or does it just look at the `CentralPackageVersionsFileImported` property?  In this case:

* `Directory.Packages.props` is imported
* `CentralPackageVersionsFileImported` == `true`
* `ManagePackageVersionsCentrally` == `false`

So Visual Studio should not update Directory.Packages.props and use the default behavior.

**Comment by jeffkl** [comment]
@tmeschter gentle ping ^

**Comment by jeffkl** [comment]
A fix for this has been submitted for the Visual Studio Project System and will be available in Visual Studio 17.6 Preview 2.

https://dev.azure.com/devdiv/DevDiv/_git/CPS/pullrequest/453924

**Comment by jeffkl** [comment]
This has been fixed and the change will be available in Visual Studio 17.6 Preview 2
